app/gui/qt/mainwindow.h

```diff
@@ -49,7 +49,6 @@ class SonicPiUDPServer;
 
 struct help_page {
   QString title;
-  QString keyword;
```

Comment:
The keyword field is still required for synths and FX that have a different keyword from their help file title for presentation or capitalization purposes.  Examples: Mod Pulse/mod_pulse, or NHPF/nhpf.  It is only redundant for help entries that aren't synths or FX.